### PR TITLE
Update review instructions

### PR DESCRIPTION
**Pull request by k-dominik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,21 +23,36 @@ Current reviewers are listed in [`bioimageio_collection_config.json`][review-con
    * request changes on resource drafts
    * upload a new version for any resource
 
-### Review Process
+### Adding a bioimage.io resource
 
-The review process technically starts after a user uploaded a _resource package_.
-Such a _resource package_ could e.g. be a newly uploaded _model package_, or _notebook package_, or an updated version of any existing resource.
+A bioimage.io resource is created by a user uploading a _resource package_.
+Such a _resource package_ could e.g. be a newly uploaded _model package_, or _notebook package_.
```
Comment (k-dominik) [inline README.md:29]: so the user is uploading a _resource package_, but the _resource package_ is already a newly uploaded _model package_, so uploading twice? ;)

File: README.md [hunk 2]
```diff
@@ -23,21 +23,36 @@ Current reviewers are listed in [`bioimageio_collection_config.json`][review-con
...
+    test-->publish
+    publish-->backup
 
 #### Staging
 
-Now the stage workflow needs to be dispatched.
-If the resource package was uploaded via the bioimage.io website, this is initiated automatically.
-In case of a url to a resource package, the `stage` workflow needs to be [dispatched manually, or via github api][staging-w] ("run workflow")[^1].
+Given a download URL to a _resource package_ the `stage` workflow needs to be [dispatched manually, or via github api][staging-wf] ("run workflow")[^1].
+If the _resource package_ was uploaded via the bioimage.io website, this is initiated automatically by the uploader service.
```
Comment (k-dominik) [inline README.md:52]: I'd have to common case first. I'd hope that would be via the website... but who knows.

File: README.md [hunk 3]
```diff
@@ -50,14 +65,19 @@ Staged resource drafts are automatically tested:
...
+Once the tests are completed, the uploader gets a notification via email that their draft is awaiting review; now a reviewer needs to take a look.
+
 An overview of all pending _resource drafts_ can be found at https://bioimageio-uploader.netlify.app/#/status.
 A _draft_ is identified by its concept id (`id` from the `rdf.yaml`).
 
 #### Review
 
 Reviewers should check the models for technical correctness (aided by CI, see [Testing section](#testing)) and contents/metadata of the resource.
+
+To this end it can be helpful to check the logs displayed at https://bioimageio-uploader.netlify.app/#/status/<concept id>.
```
Comment (k-dominik) [inline README.md:77]: for me the render preview would not add <concept id> but who knows... web

**Review by k-dominik — COMMENTED** [review]
looking good except for small stuff

File: README.md (hunk 3 above)
Comment (FynnBe) [reply README.md]: fixed (I guess it was the space.. I also added back ticks to mark it as invalid url that needs editing

File: README.md (hunk 1 above)
Comment (FynnBe) [reply README.md:29]: no any _... package_ is just a zip file...  ?

File: README.md (hunk 2 above)
Comment (FynnBe) [reply README.md]: change the order

**Pull request merged**